src/atoms/forms/select.test.tsx

```diff
@@ -1,5 +1,72 @@
 import React from "react";
+import { Select } from "./select";
+import faker from "faker";
+import { render, fireEvent } from "@testing-library/react";
 
 describe("Select", () => {
-    test.skip("TODO - https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/11", () => {});
+    test("when default props, renders with options", () => {
+        // Arrange
+        const expected = faker.random.word();
+        const testLabel = faker.random.word();
+        const testValue = faker.random.word();
+
+        // Act
+        const { getByText } = render(
+            <Select
+                id={expected}
+                options={[{ label: testLabel, value: testValue }]}
+                onChange={() => {}}
+            />
+        );
+
+        // Assert
+        expect(getByText(testLabel)).not.toBeNil();
+    });
+
+    test("when onChange set, calls handler upon change", () => {
+        // Arrange
+        let isChecked = false;
+        const expected = faker.random.word();
+        const handleChange = () => (isChecked = true);
+        const testLabel = faker.random.word();
+        const testValue = faker.random.word();
```

Comment:
I'm not sure if it matters here, but just a reminder - if having a space in one of these test values will cause it to break, we should leverage `TestUtils.randomWord` instead

Reply:
For this and the other test in this file spaces would not matter, but thanks for the feedback.